app.py

```diff
@@ -22,7 +22,7 @@ def predict():
     output = np.round(prediction[0], 2)

```

Comment:
GPT summary of [31d522](https://github.com/SampatSuthar/ml-demo/blob/987b36f074556c25b1852fddc40a2792ed434539/app.py#31d522fa64cb13a7179b3988d0450d97d57ae439) - [bca048](https://github.com/SampatSuthar/ml-demo/blob/c5811f2bfedeccf729f6750e2a2569e4bd6d3154/app.py#bca048bb925593525329c255555e16a0d9875d19):
Error: couldn't generate summary